doc/adr/0001-use-micro-frontends.md

```diff
@@ -0,0 +1,25 @@
+# 1. Use micro frontends
+
+Date: 2020-03-24
+
+## Status
+
+Proposed
+
+## Context
+
+Having the entire frontend in a monolith is the opposite of the backend architecture, where services are optional, and
+can be modified and replaced as needed. This makes it hard, if not impossible, to change or replace a part of the
+presentation layer without having to fork/throw away the whole thing.
+
+## Decision
+
+We will create separate services for individual pages and groups of pages, that will be exposed through a proxy to
```

Comment:
individual pages _or_ groups of pages?

Reply:
How about

```suggestion
We will create separate services for individual, and groups of, pages, that will be exposed through a proxy to
```

Reply:
How about https://github.com/libero/publisher/pull/387#discussion_r397678289?

Reply:
(Can't turn a single-line into a multi-line suggestion unfortunately.)

Reply:
:+1: